### PR TITLE
Added conv1d_transpose layer to paddle backend

### LINKED ISSUE #17147: conv1d_transpose

### PR DESCRIPTION
**Pull request by VaishnaviMudaliar**
Closes #17147

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by VaishnaviMudaliar** [comment]
I have made the suggested changes, let me know if I am on the right track 😊

**Comment by MahmoudAshraf97** [comment]
Hello, please check this failing test on tf frontend as it's failing because of this PR:
`ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py::test_tensorflow_conv1d_transpose`

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]
Hello, please check this failing test on tf frontend as it's failing because of this PR:
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py::test_tensorflow_conv1d_transpose

**Comment by VaishnaviMudaliar** [comment]
Could you please help me figure out what is going wrong, i tried making changes to the code but the PR run is still failing 😕

File: ivy/functional/backends/paddle/layers.py [hunk 1]
```diff
@@ -104,7 +104,19 @@ def conv1d_transpose(
...
+        output_shape = _output_shape(
+        x.shape, filters.shape, output_shape, strides, padding, 1, dilations
+    )
+    res = tf.nn.conv1d_transpose(
+        x, filters, output_shape, strides, padding, "NWC", dilations
+    )
+    if data_format == "NCW":
+        res = tf.transpose(res, (0, 2, 1))
+    return res
+    #raise IvyNotImplementedException()
```
Comment (sladyn98) [inline ivy/functional/backends/paddle/layers.py:119]: The code doesn't have error handling for when `data_format` is not "NCW" or "NWC". You might want to include some form of validation to ensure the data format is as expected. If it's neither "NCW" nor "NWC", you should raise an error or a warning.

Comment (VaishnaviMudaliar) [reply ivy/functional/backends/paddle/layers.py:119]: Thanks alot for your reply, will make the changes right away! 😊

File: ivy/functional/backends/paddle/layers.py [hunk 2]
```diff
@@ -104,7 +104,19 @@ def conv1d_transpose(
...
+    if data_format == "NCW":
+        x = tf.transpose(x, (0, 2, 1))
+        filters = tf.transpose(filters, (0, 2, 1))
+        output_shape = _output_shape(
+        x.shape, filters.shape, output_shape, strides, padding, 1, dilations
+    )
+    res = tf.nn.conv1d_transpose(
+        x, filters, output_shape, strides, padding, "NWC", dilations
+    )
+    if data_format == "NCW":
```
[the remark this answers is not in the document]
Comment (VaishnaviMudaliar) [reply ivy/functional/backends/paddle/layers.py:161]: I have made the suggested changes,please have a look :)

**Comment by sladyn98** [comment]
Thanks ! what is the reason for the failing tests

**Comment by VaishnaviMudaliar** [comment]
I am not sure either, I will look into it 🤔

**Comment by VaishnaviMudaliar** [comment]
I have made further changes to the code , as of now only 1 check is failing.

**Pull request closed without merging**